src/Stack/Config.php

```diff
@@ -51,6 +61,23 @@ public static function defineFromEnv(string $name, $defaultValue, string $envNam
         self::define($name, $value);
     }
 
+    public static function defineHostnamePortFromEnv(string $prefix, $defaultValue, $envName)
+    {
+        $value = env($envName) ?: $defaultValue;
+        $url = parse_url($value);
+
+        if ($url === false) {
```

Comment:
`On seriously malformed URLs, parse_url() may return FALSE.`....nice one PHP

Reply:
*Please*, let's just use the correct names for the configuration ENV variables.  
*Nobody* expects to pass in `host:port` when the variable name is `_HOST`.  
Let's use either `_HOST` and `_PORT` or full URIs which we can parse with the built in parsers.  
Let's also push this change upstream, if required (which it is, if I understand correctly)

Reply:
_Sorry_ @anddon , this mistake will not happen again.